### DISCUSSION TITLE
Using built-in Pkg manager

### DISCUSSION THREAD

**Discussion by Antomek** [fonsp/Pluto.jl#1249] [Questions about using Pluto]
Hello,

I am very excited about the built-in package manager feature.
Wanting to try this, I followed the instructions [here](https://github.com/fonsp/Pluto.jl/pull/844#issue-n), which just means doing
```julia
import Pluto; Pluto.run()
```
but when I try to load a package (e.g. `using Plots`) I still get:
```julia
ArgumentError: Package Plots not found in current path:

- Run `import Pkg; Pkg.add("Plots")` to install the Plots package.

    require(::Module, ::Symbol)@loading.jl:893
    top-level scope@Local: 1
```
so the built-in Pkg is not activated yet.
Have I forgotten or overlooked something?
My Pluto version is v0.14.8 and I am on Julia 1.5.

Many thanks!

**Comment by lungben** [comment]
This feature is not in the release version (0.14.x) yet. You need to use the current main branch of Pluto or wait until the V0.15 release.
